### PR TITLE
Melinda rectify

### PR DESCRIPTION
**Pull request by xiangmeiLu**
Correct some typo and modified some description.

### CODE REVIEW

File: docs/cita/architecture/view.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ title: 视图
 
 * 合约维持自身的特定储存状态；
 
-* CITA内置系统合约模块，在创始块中生成，方便用户对系统进行管理。
+* CITA内置系统合约模块，在创世块中生成，方便用户对系统进行管理。
```
Comment (leeyr338) [inline docs/cita/architecture/view.md]: There is a gap between English and Chinese

File: docs/cita/configuration/chain-config.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ id: chain-config
 title: 链级配置
 ---
 
-当拿到发布件解压后，或从源码编译后，不要着急动节点，在这之前，很重要的一步就是我们需要对链进行初始化配置。
+当拿到发布件解压后，或从源码编译后，不要着急动启节点，在这之前，很重要的一步就是我们需要对链进行初始化配置。
```
Comment (leeyr338) [inline docs/cita/configuration/chain-config.md]: "不要着急启动节点" is better?

File: docs/cita/configuration/chain-config.md [hunk 3]
```diff
@@ -154,6 +154,7 @@ usage: create_cita_config.py create [-h]
 ###  `--enable_tls` 是否开通节点间通讯加密
 * 指定节点间数据是否使用 TLS (Transport Layer Security) 加密传输，不加此选项默认为不加密传输。
 * 创建链时加上此选项，会在 `test-chain/*/network.toml` 配置文件中增加 `enable = true` 和每个 peer 中 `common_name = ${chain_name}.cita` 的配置项。
+* 该配置项不属于链级配置，不是 SysConfig 中的配置项，而是微服务 network 的配置项，在创建链时可以对它进行配置，也可以在链启动后对其进行更改。
```
Comment (leeyr338) [inline docs/cita/configuration/chain-config.md]: 最好不建议如此，在这种场景下业务一定会有中断。

File: docs/cita/system/contracts.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ CITA 链生成时，通过系统合约来生成创世块，并作为链的最基
 
 ### 节点管理
 
-按照快速搭裢的步骤，生成的链默认包含四个节点。如果你需要增加或是删除节点的话，管理员可以通过发送交易来做自定义配置。
+按照快速搭裢的步骤，生成的链默认包含四个节点。如果你需要增加或是删除普通节点的话，管理员可以通过发送交易来做自定义配置。
```
Comment (leeyr338) [inline docs/cita/system/contracts.md]: 通过发送交易管理的，应该是 “共识节点” 吧？

File: docs/cita/system/permission.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ $ bin/cita create \
 	--contract_arguments SysConfig.checkCallPermission=true SysConfig.checkSendTxPermission=true SysConfig.checkCreateContractPermission=true
 ```
 
-其中 `checkCallPermission`, `checkSendTxPermission`, `checkCreateContractPermission` 分别为合约调用、发送交易及创建合约的开关。
+其中 `checkCallPermission`, `checkSendTxPermission`, `checkCreateContractPermission` 分别为合约调用、发送交易及创建合约的开关。请注意这里的 `checkCallPermission` 与 PRC 中的 Call 并不同， PPC 中的 Call 是指对链上数据的查询，是读数据而非写数据。而这里的`checkCallPermission` 是指通过发送交易的方式调用合约接口，是指写数据的操作。
```
Comment (leeyr338) [inline docs/cita/system/permission.md]: “PRC， PPC”  -> JSON-RPC is better.

File: docs/cita/configuration/chain-config.md (hunk 2 above)
Comment (kaikai1024) [reply docs/cita/configuration/chain-config.md]: Just `当拿到发布件解压后，或从源码编译后，首先需要对链进行初始化配置。`

File: docs/cita/configuration/chain-config.md (hunk 3 above)
Comment (xiangmeiLu) [reply docs/cita/configuration/chain-config.md]: 我主要想表达的是，这个配置项现在放在了“链级配置”章节中，但是它其实不是 sysconfig的一项，而是network 的一项配置，所有想特别说明一下，如果说明欠妥，请更改！

File: docs/cita/configuration/chain-config.md (hunk 2 above)
Comment (xiangmeiLu) [reply docs/cita/configuration/chain-config.md]: 我对自己的这次 PR 中出现的大规模笔误也是醉了，

File: docs/cita/system/contracts.md (hunk 4 above)
Comment (xiangmeiLu) [reply docs/cita/system/contracts.md]: 我之前问过贤亮，如果是共识节点的增删，不能仅通过发送交易来进行配置，而要修改节点的配置文件。如果是增删只读节点，则可以用发交易来完成。 我想表达的是这个意思。

Comment (kaikai1024) [reply docs/cita/system/contracts.md]: 增删只读节点只要修改配置就好了。共识节点需要发交易

Comment (xiangmeiLu) [reply docs/cita/system/contracts.md]: 了解了！ 谢谢！

Comment (leeyr338) [reply docs/cita/system/contracts.md]: 而且现在是 P2P ，加只读节点，连配置都不需要修改。

File: docs/cita/configuration/chain-config.md (hunk 3 above)
Comment (leeyr338) [reply docs/cita/configuration/chain-config.md]: 其实不需要说明，是否加密本身就是链的一个属性。只不过，它是放在了 `network` 这个模块承载实现而矣。

File: docs/cita/architecture/view.md (hunk 1 above)
Comment (boundless-forest) [reply docs/cita/architecture/view.md]: 英文单词左右两侧应该加个空格的。 ` CITA 内置系统合约模块`

File: docs/cita/configuration/chain-config.md (hunk 3 above)
Comment (xiangmeiLu) [reply docs/cita/configuration/chain-config.md]: 这一章节的开头有一句 “这些配置信息将被写入链的创世块，创世块一旦生成，SysConfig 中只有 `chainName`，`operator`，`website` 这三项可以在链运行之后再进行修改，其他项均不可再修改, 因此请大家慎重设定各配置项。” --  所以我才特意要在这里说明的。

**Comment by chaoticlonghair** [comment]
Typo in commit messages:

image

Never mind, it's unimportant.

**Comment by kaikai1024** [comment]
> Never mind, it's unimportant.

Then you can approve it.

BTW, it's duplicate after fix the typo.

**Comment by chaoticlonghair** [comment]
> Then you can approve it.

Prove is more difficult than find bugs.
Even I can't find more bugs, it doesn't mean there is no bugs.

So, IMHO, **DO NOT** approve PRs too quick.

(In the follow paragraph, I am not talk about this PR, I mean some other PRs.) 
Sometimes, approving too quick doesn't mean the reviewer has high efficiency and he/she works hard， it just proves how ignorant (or stupid) he/she is.

**Comment by kaikai1024** [comment]
> DO NOT approve PRs too quick.

Should `DO NOT merge PRs too quick`.

> Even I can't find more bugs

if you can't find more bugs. you can approve it. you approve it does not mean it can merge.(though it's not like this now. but the process of reviewing is ok: if you review and you think it's ok, then you can approve it and you should be responsible for the code. maybe we can discuss how to be responsible for the code.)

> Then you can approve it.

If you approve it, i'll dismiss it. i think it's not unimportant.
You should follow the rules of the contributions if you want contribute.
the commits is absolutely not ok in CITA. but this repo i'm not sure.

> approving too quick doesn't mean the reviewer has high efficiency

yes, fuck this behavior. the reviewer should be serious about the review instead of just to merge the pr.

**Comment by kaikai1024** [comment]
And i think the one who merge the pr should be more responsible for the pr instead of the reviewer. so if you believe the reviewer, you can merge it and accept the punishment if there's something wrong about this pr.

**Review by leeyr338 — DISMISSED** [review]

**Review by kaikai1024 — APPROVED** [review]

**Pull request merged**